config/config.go

```diff
@@ -34,8 +34,14 @@ func (cfg Config) String() string {
 
 // New returns a default Config overridden with given parameters.
```

Comment:
We may want to update the doc to state that this returns something that may not be valid/usable and that its completely fine as it is meant to be merged before anything else.